ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.16.yaml

```diff
@@ -61,15 +61,27 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-ipi-shared-vpc-phz-sts-fips-f14
-  cron: 14 13 8,22 * *
+- as: aws-ipi-localzone-byo-subnet-ovn-f28
+  cron: 19 0 4 * *
+  steps:
+    cluster_profile: aws-1-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FEATURE_GATES: '[InstallAlternateInfrastructureAWS=true]'
+      FEATURE_SET: CustomNoUpgrade
```

Comment:
a bit late since already merged, but isn't this feature gate interferes with cluster upgrade? 
@jiajliu I recall you're familiar with this one right? does it have the same effect as `TechPreviewNoUpgrade`

Reply:
Good catch, I commented it on another line in https://github.com/openshift/release/pull/50805#discussion_r1568542486, @JianLi-RH can you file another PR to remove it.

Reply:
@evakhoni from api side, I guess `CustomNoUpgrade` should work the same with `TechPreviewNoUpgrade`, which means it will block a minor version update, unblock a z-stream update. But there is little description about `CustomNoUpgrade` in official doc. 

cc @wangke19 do u know more about `CustomNoUpgrade`?

Reply:
so its blocking only minor upgrade? by means of upgradeable=false similar to admin-gates? in this case its only in `4.16:stable->nightly` which is z-stream so probably still expected to pass :thinking:

Reply:
Searched [log](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-localzone-byo-subnet-ovn-f2/1782303741081292800/artifacts/aws-ipi-localzone-byo-subnet-ovn-f2/gather-extra/artifacts/clusterversion.json) here. From the condition, it should pass on z-stream update.

```
                    {
                        "lastTransitionTime": "2024-04-22T07:13:58Z",
                        "message": "Cluster operator config-operator should not be upgraded between minor versions: FeatureGatesUpgradeable: \"CustomNoUpgrade\" does not allow updates",
                        "reason": "FeatureGates_RestrictedFeatureGates_CustomNoUpgrade",
                        "status": "False",
                        "type": "Upgradeable"
                    },
```